### ISSUE TITLE
Hello, pastboard how to listen the ctrl + v event on desktop?

### ISSUE THREAD

**Issue by lucasjinreal** [MixinNetwork/flutter-plugins#312]
So that i can get image bytes from pastboard

**Comment by boyan01** [comment]
https://api.flutter.dev/flutter/services/HardwareKeyboard-class.html

https://api.flutter.dev/flutter/widgets/FocusableActionDetector-class.html

**Comment by lucasjinreal** [comment]
This is not what I want.

I want textfield able to ctrl + v paste image on desktop, since your repo looks like provides this functionality, it could be benificial for all flutter delveopers if there could be some dicuss on how to intergrate it with textfiled widget.

**Comment by boyan01** [comment]
There is no official way to handle ctrl +  v on textfield

but you can see the code below

https://github.com/MixinNetwork/flutter-app/blob/fb40a8be9fcc97590992e4f35f8762b311774587/lib/ui/home/chat/input_container.dart#L472

https://github.com/MixinNetwork/flutter-app/blob/fb40a8be9fcc97590992e4f35f8762b311774587/lib/ui/home/chat/input_container.dart#L974

**Comment by lucasjinreal** [comment]
@boyan01 thanks for the suggestion, your app looks exactly what I want achieve, but your code some sort to high level hard to understand.

Many cusotmized widget there, I just saw there is a PasteTextIntent: _PasteContextAction(context), Is there anyway to make it work with textField, as simple as possible.

BTW, I downloaded your app, it's looks very nice! Would consider make your ChatUI be a seperated libbrary?
